openpower/package/openpower-pnor/openpower-pnor.mk

```diff
@@ -4,12 +4,12 @@
 #
 ################################################################################
 
-OPENPOWER_PNOR_VERSION ?= 5181154715dd12e10f1d1ac432e128f44e8d32c0
-OPENPOWER_PNOR_SITE ?= $(call github,open-power,pnor,$(OPENPOWER_PNOR_VERSION))
+OPENPOWER_PNOR_VERSION ?= c595a446b766088770b46144baf720c52d8eec36
+OPENPOWER_PNOR_SITE ?= $(call github,jahunsbe,pnor,$(OPENPOWER_PNOR_VERSION))
```

Comment:
Will need to get your changes merged in the openpower-pnor repo and move this back to open-power and update the version.